de.dlr.sc.virsat.external.lib.lyo/about_files/THIRD-PARTY.txt

```diff
@@ -0,0 +1,66 @@
+
+Lists of 64 third-party dependencies.
+     (Apache 2.0) Jackson-annotations (com.fasterxml.jackson.core:jackson-annotations:2.10.1 - http://github.com/FasterXML/jackson)
+     (Apache 2.0) Jackson-core (com.fasterxml.jackson.core:jackson-core:2.10.1 - https://github.com/FasterXML/jackson-core)
+     (Apache 2.0) jackson-databind (com.fasterxml.jackson.core:jackson-databind:2.10.1 - http://github.com/FasterXML/jackson)
```

Comment:
Jackson dependencies are already introduced via the external lib jersey. should check if we need them here again, or if we have to externalize it.